test/extended/testdata/cmd/test/cmd/builds.sh

```diff
@@ -44,14 +44,14 @@ os::cmd::expect_success_and_text "oc get bc/centos --template '${template}'" '^D
 os::cmd::expect_success 'oc delete is/centos bc/centos'
 
 # Build from Dockerfile with given output ImageStreamTag spec
-os::cmd::expect_success "oc new-build -D \$'FROM centos:7\nENV ok=1' --to origin-test:v1.1"
-os::cmd::expect_success_and_text "oc get bc/origin-test --template '${template}'" '^ImageStreamTag origin-test:v1.1$'
+os::cmd::expect_success "oc new-build -D \$'FROM centos:7\nENV ok=1' --to origin-test:v3.11.0"
+os::cmd::expect_success_and_text "oc get bc/origin-test --template '${template}'" '^ImageStreamTag origin-test:v3.11.0$'
 
 os::cmd::expect_success 'oc delete is/centos bc/origin-test'
 
 # Build from Dockerfile with given output DockerImage spec
-os::cmd::expect_success "oc new-build -D \$'FROM centos:7\nENV ok=1' --to-docker --to openshift/origin:v1.1-test"
-os::cmd::expect_success_and_text "oc get bc/origin --template '${template}'" '^DockerImage openshift/origin:v1.1-test$'
+os::cmd::expect_success "oc new-build -D \$'FROM centos:7\nENV ok=1' --to-docker --to openshift/origin:v3.11.0-test"
+os::cmd::expect_success_and_text "oc get bc/origin --template '${template}'" '^DockerImage openshift/origin:v3.11.0-test$'
```

Comment:
i don't see why this file had to change?

Reply:
@bparees I was suprised also to see the tests failing prior to this change; having a look at docker hub (website) we got non working link for this versions. 
And trying to pull origin/openshift:v1.1 image, we got an error.

I was thinking that docker kind of archived or pruned too old images.

Reply:
1) this is the to, so it's pushing, not pulling that image
2) this test doesn't care if the build succeeds, only if we can create it.

do you have an example of the failure you were hitting?

Reply:
(running these commands locally against a cluster they seem to succeed)

Reply:
my bad @bparees , I read `FROM openshift/origin:v1.1 `  and I advised to do the change. We can definetly revert this change.